weld/subsystem/src/main/java/org/jboss/as/weld/services/bootstrap/WeldSecurityServices.java

```diff
@@ -94,6 +94,10 @@ public void cleanup() {
 
     @Override
     public org.jboss.weld.security.spi.SecurityContext getSecurityContext() {
+        if (securityManagerSupplier == null) {
```

Comment:
So there seem to be only one case when this can be invoked - from within `SecurityServices#getSecurityContextAssociator()` when `elytronDomation` is null. Can this even be a case? E.g. can you at any time have neither elytron nor picketbox on CP?

If so, then this is ok, I am just asking since my knowledge of security has the quality of a swiss chesse - it has lots of holes :)

Reply:
It gets hard to achieve due to default security domain attributes on some subsystems but yes it should be possible that a deployment is deployed with no security activated so there could be a case legacy security is not available and no Elytron domain is associated either.

Reply:
Ok, thanks for quick clarification.